tests/CalendarHTMLFormatterTest.php

```diff
@@ -1,33 +1,35 @@
 <?php
 
-namespace CultuurNet\CalendarSummaryV3;
+namespace CultuurNet\CalendarSummaryV3\Tests;
 
+use CultuurNet\CalendarSummaryV3\CalendarHTMLFormatter;
+use CultuurNet\CalendarSummaryV3\FormatterException;
 use CultuurNet\SearchV3\ValueObjects\Event;
+use PHPUnit\Framework\TestCase;
 
-class CalendarHTMLFormatterTest extends \PHPUnit_Framework_TestCase
+class CalendarHTMLFormatterTest extends TestCase
 {
     /**
      * @var CalendarHTMLFormatter
      */
     protected $formatter;
 
-
-    public function setUp()
+    protected function setUp(): void
     {
         $this->formatter = new CalendarHTMLFormatter();
     }
 
-    public function testGeneralFormatMethod()
+    public function testGeneralFormatMethod(): void
     {
         $offer = new Event();
         $offer->setCalendarType(Event::CALENDAR_TYPE_SINGLE);
         $offer->setStartDate(new \DateTime('2018-01-25T20:00:00+01:00'));
         $offer->setEndDate(new \DateTime('2018-01-25T21:30:00+01:00'));
 
-        $this->formatter->format($offer, 'xs');
+        $this->assertIsString($this->formatter->format($offer, 'xs'));
```

Comment:
Is it worth also checking the actual string versus the expected string?

Reply:
This question applies to other `assertIsString` occurences.

Reply:
Not really necessary because they were already covered by other tests, but I added it now